src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -59,8 +59,8 @@ function create_config (input_dir, output_dir, hostname)
                             { vlan = vif_details.zone_vlan,
                               mac_address = port.mac_address,
                               port_id = port.id,
-                              ingress_filter = filter(port, secbindings, secrules, 'ingress'),
-                              egress_filter = filter(port, secbindings, secrules, 'egress'),
+                              ingress_filter = filter(port, secbindings, secrules, 'ingress', vif_details.packetfilter),
```

Comment:
The `packetfilter` setting should be in `profile` rather than `vif_details`.

`profile` contains configuration supplied by the user and `vif_details` contains information determined by the OpenStack code. (This is OpenStack Neutron semantics and not some funny invention of mine :-))